APM Security hub (#8017)
DataDog/documentation: content/en/tracing/security/_index.md

---

> title: Security and Fine-tuning

Reviewer: ```suggestion
title: Security and Fine-Tuning
```

kind: documentation
description: "Configure the Datadog Tracer or Agent to modify or discard spans for security or fine-tuning purposes."
aliases:
    - /security/tracing
    - /tracing/guide/security
further_reading:
    - link: 'tracing/security/agent_span_modifications'
      tag: 'Documentation'

>       text: 'Configure the Datadog Agent to modify or discard spans.'

Reviewer: ```suggestion
      text: 'Configure the Datadog Agent to modify or discard spans'
```

    - link: 'tracing/tracer_span_modification'
      tag: 'Documentation'

>       text: 'Configure the Tracer to modify or discard spans.'

Reviewer: ```suggestion
      text: 'Configure the Tracer to modify or discard spans'
```

---
## Overview

> The Datadog Agent and all tracing libraries have several options available to modify or discard spans.

Reviewer: This page should open with why people might want to do this, so they know if they're in the right place. Something like: "The performance data and traces that you're collecting with Datadog can contain sensitive information that you want to filter out, obfuscate, scrub, filter, modify, or just not collect."

> These configuration options allow for a variety of use cases related to obfuscating or removing sensitive information from traces, as well as discarding traces from certain endpoints or matching criteria. Various options are described below.  Since it is not possible for us to detail every possible configuration of the Tracer and Agent, we have illustrated common examples.

Reviewer: ```suggestion
These configuration options allow for a variety of use cases related to obfuscating or removing sensitive information from traces, as well as discarding traces from certain endpoints or matching criteria. Various options are described below.  These docs cover several common methods for configuring Tracer and Agent to achieve these security requirements.
```

> If the below examples do not cover your security or fine-tuning needs, reach out to our awesome [support team][1], who will be more than happy to provide guidance.

Reviewer: ```suggestion
If your fine-tuning needs aren't covered and you need assistance, reach out to [the Datadog support team][1].
```

> ## Agent Configurations

Reviewer: All these headings are framed in terms of what you do; it would be better if they described what the reader is trying to accomplish, for example, "Exclude some traces from being collected"
